hw01_hello_otus/go.mod

```
@@ -1,3 +1,17 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/home_work/hw01_hello_otus
```

```suggestion
module github.com/vitaLina89/test-go/hw01_hello_otus
```
Чтобы модуль был доступен для загрузки и импорта, в наименовании следует указывать полный путь до него, включая название репозитория.